.github/workflows/benchmark.yml

```diff
@@ -8,10 +8,12 @@ on:
     paths-ignore:
       - "**/*.md"
       - "LICENSE"
+      - 'examples/**'
```

Comment:
Sorry for the late comment on this - would the examples ever be so big / slow that it's worth having a separate build for them? How about just running example tests from the normal `go run mage.go test`?

Reply:
That makes sense, I'm good either way. I wanted to get ready for any type of example, but it can be an overkill

Reply:
Either way we are blocking code submission unless both pass. So I guess it's mostly the same if we bundle them to one task.